packages/gasket-plugin-webpack/index.js

```diff
@@ -6,29 +7,36 @@ const { name, devDependencies } = require('./package');
 /**
 * Creates the webpack config
 * @param  {Gasket} gasket The Gasket API
-* @param {Object} webpackConfig Initial webpack config
-* @param {Object} data Additional info
+* @param {Object} initConfig Initial webpack config
+* @param {Object} nextOptions Additional info
 * @returns {Object} Final webpack config
 */
-function initWebpack(gasket, webpackConfig, data) {
-  const { execSync, config } = gasket;
+function initWebpack(gasket, initConfig, nextOptions) {
```

Comment:
This function could be used by non-next apps to initialize webpack. If you want to change this variable name then `context` would more general and appropriate.
```suggestion
function initWebpack(gasket, initConfig, context) {
```

Reply:
Updated.